### ISSUE TITLE
Do you pre-train backbones weights on imagenet?

### ISSUE THREAD

**Issue by levyfan** [deepinsight/insightface#93]
Do you pre-train the modified Res50/... backbones weights on imagenet and then fine-tune the entire model + customized loss on face dataset?

**Comment by nttstar** [comment]
No, from scratch.
